src/routes/rules/RulesView.vue

```diff
@@ -405,10 +425,15 @@ export default {
 .text-label-lg {
   margin-bottom: 8px;
 }
-
+@media (max-width: 600px) {
+  .section-title {
+    padding-left: 0px;
+    text-align: center;
+  }
+}
```

Comment:
I like the look of this change, but I fear it makes the headers look to much like big buttons, since we use the same color and center-align the text in our buttons
<img width="500" alt="image" src="https://github.com/cuttle-cards/cuttle/assets/6520704/eb82a600-b90e-4646-8e55-caa409cf4d0d">


```suggestion
```